controllers/api.controller.js

```diff
@@ -0,0 +1,10 @@
+const { showAllEndpoints } = require("../models/api.model");
+exports.getEndPoints = (req, res) => {
+  showAllEndpoints()
```

Comment:
It's fine that you've gone down the readFile route but, alternatively, you could require in the data in this controller file and then just send it back (no need for a model in that situation)